lib/glific/users/user.ex

```diff
@@ -45,7 +45,8 @@ defmodule Glific.Users.User do
           last_login_from: String.t() | nil,
           upload_contacts: boolean() | false,
           consent_for_updates: boolean(),
-          confirmed_at: :utc_datetime | nil
+          confirmed_at: :utc_datetime | nil,
+          organization_name: String.t() | nil
         }
```

Comment:
We need to add the organization name inside the `trial_metadata` column in users table, since we already have a `name` column in the orgs table and this field is specific to trial accounts.
can refere how we are storing the status of the trial user here: `lib/glific/saas/queries.ex:283`.